README.md

```diff
@@ -397,6 +397,31 @@ You can specify scripts to execute Cloud Functions with the `scripts` option:
 ]
 ```
 
+You can also specify custom fields with the `scrips` option:
+
+```json
+"apps": [
+  {
+    "scripts": [
+      {
+        "title": "Delete account",
+        "classes": [
+          {
+            "name": "_User",
+            "fields": [
+              { "name": "createdAt", "validator": "value => value > new Date(\"2025\")" }
+            ]
+          }
+        ],
```

Comment:
Now that I see this syntax side by side, it seems to be a change in in field value type. The value for key `classes` is suddenly not `Array<String>` any more but can also be `Array<Object>`. If we released the scripts feature only in alpha and/or beta version we could make this breaking change and don't have to support `Array<String>` anymore to simplify code.

Reply:
Leaving as is